11-serialization-with-json/projects/final/lib/ui/recipes/recipe_list.dart

```diff
@@ -139,7 +140,7 @@ class _RecipeListState extends State<RecipeList> {
         child: Row(
           children: [
             IconButton(
-              icon: Icon(Icons.search),
+              icon: Icon(Icons.search,),
```

Comment:
Is this a typo? The same is in the starter project

Reply:
Nope. Just a few differences in my final project and these projects. Trying to get them to be the same

Reply:
Ok but is this comma needed?
```
Icons.search,),
            ^
```

Reply:
No. Just common to add commas after everything for formatting

Reply:
Probably should remove it, but then would have to remove it everywhere

Reply:
Oh ok. Makes sense. Let's keep it as is.

Reply:
Based on my convo with @orionthewake the formatting comma should be included. This puts the code in line with how Google formats the code.